packages/react/src/grid/PageGrid.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { PropsWithChildren } from 'react'
+
+export type PageGridColumnNumber = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12
+
+export type PageGridProps = PropsWithChildren<{}>
+
+export const PageGrid = ({ children }: PageGridProps) => <div className="amsterdam-page-grid">{children}</div>
```

Comment:
I’ve called it `PageGrid` because I expect a separate `Grid` component in the future with more props.

This component will always produce the full-page grid of 1440 (1920) pixels, 4 or 12 columns, etc.

Reply:
Could it have other children than `GridCell`? If they are closely related then having it as a sub component might be an option. This does not restrict the client from providing other elements by the API is more clear.
```
<PageGrid>
  <PageGrid.Cell>
  <PageGrid.Cell>
</PageGrid>

Reply:
While I like that pattern in general, I feel we’ll have multiple `*Grid` components, but just one `GridCell` component. Plus, we’ll allow all components that implement `GridCellProps` as children of `PageGrid` and other grid components. So I’ll leave it like this for now.